tests/test_matrix.py

```diff
@@ -188,34 +223,3 @@ def test_multiplyMatrix():
     # assert matPro.__str__() == ""
     """
     pass
-
-
```

Comment:
I think you accidentally removed this code (lines 198-221 in test_matrix.py) while resolving the merge conflict. Please add these lines again. Also, delete the log.txt file and add it to .gitignore file.

Reply:
@8hantanu Yes, I intentionally removed lines 198 - 221 because they contained the test cases for isSquare and transposeMat, where there were minor errors. So, I fixed those errors, and moved those functions up under the "matrix.structure" label, just below test_traceMat() function (check line 33-51 in my latest commit). All the check functions work fine now.